term/patterns/social-in-content.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Title: Social in content
+ * Slug: term/social-in-content
+ * Categories: text
+ * Block Types: core/social-links
```

Comment:
we only add this when the pattern is styling just one block, mainly blocks that have variations like the query block, so we can remove this line in this case

Reply:
Removed in latest commit